packages/foundry/src/replaceTokens.ts

```diff
@@ -0,0 +1,32 @@
+import { join } from "node:path";
+import { readFile, writeFile } from "node:fs/promises";
+import { glob } from "glob";
+
+export async function replaceTokens(globPatterns: string[], values: Record<string, string>, outputDirectory: string) {
+    if (!globPatterns || globPatterns.length === 0 || !values || Object.keys(values).length === 0) {
```

Comment:
It's not a public API, you can assume that there will always be a globPattern provided since it's a required argument. You do not need to validate the input.